process.py

```diff
@@ -0,0 +1,24 @@
+MAJOR_COLORS = ['White', 'Red', 'Black', 'Yellow', 'Violet']
```

Comment:
File names are generic. It should indicate or provide an hint about the contents and what is it about that the file contains.
Naming can be improved...